qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -36,17 +36,23 @@ class BasisGateLibrary(ABC, Mapping):
     # Location where default parameter values are stored. These may be updated at construction.
     __default_values__ = {}
 
+    # Parameters that do not belong to a schedule, a set of names
+    __parameters_without_schedule__ = set()
+
     def __init__(
         self,
         basis_gates: Optional[List[str]] = None,
         default_values: Optional[Dict] = None,
+        params_without_schedule: Optional[Set] = None,
```

Comment:
Do we really need this constructor argument? Current implementation, i.e. constructor of `Calibrations`, doesn't take parameter names, so I think this is sort of new feature. I don't think we have a use case we need to add new parameters.